<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <!--[if lt IE 9]>
    <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
  <![endif]-->
  <link rel="stylesheet" href="./stylesheets/Mystyle.css">
</head>
<body>
<h3 id="recognizing-relatives-and-ancestors-analogies-homologies">Recognizing relatives and ancestors: analogies &amp; homologies</h3>
<hr />
<p>Well before the modern theory of evolution was proposed, it was obvious to naturalists, and particularly the Swedish naturalist <a href="http://www.ucmp.berkeley.edu/history/linnaeus.html">Carl Linneaus (1707-1778)</a>, that various organisms could be classified into a hierarchical scheme (you may have heard of it: kingdoms, phyla, orders, families, etc).</p>
<p>In this scheme, each type of organism is given a unique place and a unique name. The most natural of these groups are “species”, which are defined by the fact that organisms that are of the same species can (and do) successfully interbreed with one another. </p>
<p>The <a href="http://plato.stanford.edu/entries/species/">concept of a species</a> is complex, in large part because populations vary and evolve over time. The biological species concept was defined by Mayr. </p>
<blockquote>
<p>Species are groups of actually (or potentially) interbreeding natural populations which are reproductively isolated from other such groups.” - Ernst Mayr (1953)</p>
</blockquote>
<p>This definition is not completely unambiguous. For example, when, over the course of time does one species become distinct from its progenitor?  How do you define a species when individuals do not interbreed. </p>
<p>A particularly interesting example is known as a <strong>ring species.</strong> Imagine populations of the species A; over the geographic range of A where there exist a number of subpopulations. </p>
<p>These subpopulations (A<sub>1</sub> to A<sub>5</sub>) and (A<sub>a</sub> to A<sub>e</sub>) have limited regions of over-lap with one another, but where they overlap they interbreed successfully.</p>
<p>But, populations A<sub>5</sub> and A<sub>e</sub> no longer interbreed successfully – are these populations separate species?  </p>
<p><a href="http://www.actionbioscience.org/evolution/irwin.html"><img src="./img/ring_species.gif" alt="ring" /></a></p>
<p><em>Figure: Ring species. Click the image to learn more</em></p>
<p><strong>Defining relationships between species morphologically:</strong>The linnean classification system suggests a tree-like organization to the relationships between organisms. </p>
<p>How do we arrange organisms on such a phylogenic tree?</p>
<p>One approach is known as <a href="http://www.fossilnews.com/1996/cladistics.html">cladistics</a>. Each type of organism is analyzed in terms of primitive (ancestral) and derived (that is, more specialized) characteristics. </p>
<p>Based on these traits, organisms can be arranged into <a href="http://www.amnh.org/exhibitions/Fossil_Halls/cladistics.html">trees.</a></p>
<p>Of course, exactly how such a tree is structured depends upon which traits are considered to be primitive (and so present in ancestral forms) and which traits are considered to represent recent specializations<a href="http://www.amnh.org/exhibitions/Fossil_Halls/cladistics.html">.</a><br /> <img src="./img/lineageA.jpg" alt="lineage A" /><img src="./img/lineageB.jpg" alt="lineageB" /></p>
<p>The ability to determine the complete sequence of an organism’s DNA has made it possible to use other traits, DNA sequence and genome organization, to study the relationship of organisms with respect to one another.</p>
<p>In most cases this has confirmed anatomical based classification schemes, while in others it has revealed unexpected relationships.   </p>
<hr />
<p><strong>Classification beyond species: </strong></p>
<p>The next higher level of classification is “genus”.  Organisms that are similar to one another, but clearly distinct, can be placed in the same genus.  For example there are a number of species in the genus <em>Homo</em> [<a href="http://en.wikipedia.org/wiki/Human_evolution">link</a>], these include <em>sapiens</em> (modern people), <em>floresiensis</em>, <em>habilis,</em> and <em>neanderthalensis</em>. </p>
<p><a href="http://www.pandasthumb.org/pt-archives/000582.html"><img src="./img/img/flor_and_sapiens.gif" /></a></p>
<p>Given their similarities, we assume that all members of a particular genus share a common ancestor in the relatively recent (millions to tens of millions of years) past. </p>
<p>As we look deeper at the anatomical and molecular structure of organisms, particularly their genomic DNA sequences, we find evidence for deeper relationships.</p>
<p>Humans, are grouped together with a number of living organisms (lemurs, lorises, tarsiers, new world monkeys, old world monkeys, and apes) and extinct species in the order of <a href="http://tolweb.org/treehouses/?treehouse_id=3029">Primates</a> [<a href="http://tolweb.org/Primates/15963">link</a>]. </p>
<p>The available evidence indicates that all primates shared a common ancestor that lived approximately 85 million years ago (that is about 20 million years before the dinosaurs went extinct!). </p>
<figure>
<img src="./img/human-linnean-classification.jpg" alt="human classification" /><figcaption>human classification</figcaption>
</figure>
<p>The primates are divided into two suborders, all members of one of these, the Haplorrhini, which includes tarsiers, monkeys, apes, and human beings share a requirement for vitamin C in their diet.  This requirement is due to a mutation in a gene (known as GULO or 1-gulono-gamma-lactone oxidase). GULO1 is required for the synthesis of vitamin C.  This mutation in the ancestor or the Haplorrhini appears to have occurred ~40 million years ago.</p>
<p><a href="http://anthro.palomar.edu/earlyprimates/first_primates.htm"><img src="./img/VitaminCPhylogeny.jpg" alt="vitamin C" /></a></p>
<p>Putting a working copy of this gene, <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=14962674&amp;query_hl=16&amp;itool=pubmed_docsum">derived form a mouse</a>, into human cells “cures” them of the need for exogenous vitamin C.</p>
<p>Guinea pigs also require vitamin C, but since i) there are primates, such as lemurs, that do not require vitamin C and ii) since the common ancestor of primates and guinea pigs lived more than 80 million years ago, it appears that the mutations that make Haplorrhini primates and guinea pigs vitamin C dependent occurred independently.</p>
<p>Now the deeper question is, why should it be possible to group organisms into such a hierarchical system.  Why should we share traits, at both the anatomical and DNA sequence level with lemurs and tarsiers? </p>
<p>Such an ordering of organisms is difficult to understand if each organism originated independently - if each were created de novo.</p>
<p>On the other hand it makes complete sense if each modern organism evolved from an ancestor, and that that ancestor gave rise (over evolutionary time) to multiple species. The reason that organisms appear related is that they <strong>are</strong>, they share a common ancestor. </p>
<p>Here it is worth noting quite explicitly that while we are primates, no living primate is the ancestor to humans.  What we share is a common ancestor that lived over 80 million years ago. </p>
<hr />
<p>Phylogenetic trees are generally compared based on the principle of parsimony (<a href="http://en.wikipedia.org/wiki/Occam%27s_Razor">Occam’s razor</a>.) </p>
<p>This principle states that, all else being equal, the simplest model that explains the data is preferable to a more complex one. When two models produce the same predictions and explain the same set of data, we prefer the simpler of the two.</p>
<p>In the case of the relationships between organisms, evolutionary mechanisms provide the simplest, and most predictive explanation. </p>
<p><strong>Questions to answer</strong></p>
<ol type="1">
<li>How would you go about determining whether an organism, identified through fossil evidence, was part of a new or a living species?  </li>
<li>How would you determine whether two species are part of the same genus?</li>
<li>Why is the common need of primates for vitamin C evidence for a common ancestor?</li>
<li>Why do we assume that the mutations in Haplorrhini primates and guinea pigs were independent events?</li>
<li>How could vitamin C dependence have arisen in the first place?</li>
<li>What types of traits are likely to be primitive?</li>
<li>How can you decide whether a trait is primitive or derived? Is it always unambiguous?</li>
</ol>
<p><strong>Questions to ponder</strong></p>
<ul>
<li>How does a species differ from a genus or an order, are both equally “natural” groupings? </li>
<li>What predictions does creation science/intelligent design make about phylogenic relationships?</li>
</ul>
<hr />
<p><strong>Homologs and Analogs</strong></p>
<p>A major problem in classifying organisms is deciding whether similar traits were inherited from a common ancestor or whether they represent independent solutions to a particular problem.</p>
<p>For example, an anatomical analysis of the forelimb of mammals suggests that they are <strong>homologous</strong> structures.</p>
<p>To say that two structures are homologous means that they are derived from a common ancestor and that the feature was present in that common ancestor.</p>
<p>This type of reasoning leads to the prediction that fossils of the last common ancestor of the mammals will be found to have a forelimb with a structure similar to that of modern mammals.</p>
<p><a href="http://www.pbs.org/wgbh/evolution/library/04/2/l_042_01.html"><img src="./img/TetrapodLimb.jpg" /></a></p>
<p>Not all similar structures are homologous, however. The alternative is that the structures are <strong>analogous</strong>. When two structures are analogous, it means that they evolved independently, but perform the same functions or have a similar structure.</p>
<p>Their common form was determined by their common function.</p>
<p>Consider the wing of a pterodactyl, a flying reptile, a bird, and a bat, a mammal.</p>
<p><a href="http://www.nurseminerva.co.uk/adapt/wings.htm#bat"><img src="./img/Wings.jpg" /></a></p>
<p>Analysis of these structures indicates that they are distinctly different from one another.</p>
<p>In the pterodactyl, the wing membrane is supported by the 5th finger of the forelimb, in the bird by the 2nd finger, and in the bat, by the 3rd, 4th and 5th fingers.</p>
<p>The last common ancestor of flying reptiles,birds and mammals did not have wings, although it certainly had forelimbs.</p>
<p>While wings of pterodactyls, birds, and bats are analogous structures, their forelimbs are <strong>homologous</strong>.</p>
<p>The physics of flight are constant. Organisms of similar size face the same aerodynamic and thermodynamic constraints. In general there are only a limited number of physically workable solutions to deal with these constraints.</p>
<p>Under these conditions different populations will, through the process of variation and selection, end up with structurally similar solutions. This process is known as convergent evolution. </p>
<p>Convergent evolution occurs when only certain solutions to a particular problem are evolutionarily accessible.  As another example, the use of a dagger is an effective solution to the problem of killing another organism.</p>
<p>Variations of this solution have been “discovered” or invented independently many times.</p>
<p>Marsupial and placental mammals diverged from a common ancestor approximately 160Myr ago.</p>
<p>Since then, they have converged on a number of similar adaptations.</p>
<embed width="420" height="345" src="http://www.youtube.com/v/ntVI4N7AUx0" type="application/x-shockwave-flash"> </embed>

<hr />
<p><strong>The loss of traits</strong></p>
<p>Another process that can confuse or complicate cladistic analysis is the loss of traits. </p>
<p>As organisms adapt to a specific environment and life style, traits once useful can become irrelevant and may be lost.</p>
<p>A classic example is the reduction of hind limbs during the <a href="http://ngm.nationalgeographic.com/ngm/data/2001/11/01/html/ft_20011101.4.html?fs=animals-panther.nationalgeographic.com">evolution of whales</a>. </p>
<p><a href="http://www.pbs.org/wgbh/evolution/library/03/4/l_034_05.html"><img src="./img/whale_hindlimb.gif" /></a></p>
<p>Another common loss is that of eyes, often seen as populations adapt to environments in which light is absent. The loss of traits can itself be an adaptation if it provides an advantage to organisms living in a particular environment. </p>
<p><a href="http://www.aloha.net/%7Esmgon/eyes.htm"><img src="./img/eyeless_trilobites.jpg" /></a></p>
<p>This fact can make it difficult to determine whether an organism is primitive (that is retains ancestral features) or is highly evolved. </p>
<p><strong>Evidence for a common ancestor:</strong> Similar structures may arise independently, while apparently different structures, such as a human hand and a whale’s flipper, are related.</p>
<p>How do we distinguish between these two possibilities?  In general, we carry out a detailed comparative analysis of the development and final form of the structure. </p>
<p>In the modern world of molecular biology, this involves the genes that are active as the structure forms. </p>
<p>The more details two structures share, the more likely they are to be homologous.</p>
<p><a href="http://i.imgur.com/emPJ3.gif"><img src="./img/phylogenetic-tree-and-time.gif" alt="phylo" /></a></p>
<p>Detailed analysis of many different types of organisms reveals the presence of a common molecular signature that strongly suggests that all living organisms are closely related; that is, that they share a common ancestor.</p>
<p>What are these apparent universal homologies? They range from the basic structure of cells to the molecular machinery involved in energy capture and transduction, information storage and utilization. </p>
<p>To review (superficially), all extant organisms …</p>
<ul>
<li>use double stranded DNA as their genetic material</li>
<li>use the same molecular systems, transcription and translation, to access the information stored in DNA</li>
<li>use a common genetic code, with few variations, to specify the sequence of polypeptides (proteins)</li>
<li>use ribosomes to translate the information stored in messenger RNAs into polypeptides</li>
<li>share common enzymatic (metabolic) pathways</li>
</ul>
<hr />
<p><strong>Questions to answer</strong></p>
<ol type="1">
<li>What typical mammalian traits have whales lost during their evolution? which have they retained?</li>
<li>How does the loss of a trait complicate a cladistic analysis? </li>
<li>Does the premise of “intelligent design” make any clear or testable predictions about the apparent relatedness of different species? Is it useful for understanding homologous and analogous structures?</li>
<li>Is convergent evolution possible at the molecular level?  </li>
<li>If all organisms are descended from a common ancestor, what can we say about the diversity of pre-biogenic systems that existed before that ancestor?</li>
</ol>
<p><strong>Questions to ponder</strong></p>
<ul>
<li>How does the presence of convergent evolution complicate phylogenic analyses?  What general types of conditions might lead to a complex organism becoming simpler?</li>
<li>What benefit(s) could be linked to the loss of eyesight or other “advanced” traits? </li>
</ul>
<hr />
<script language="javascript" type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js"></script>
<script language="javascript" type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.9.2/jquery-ui.min.js"></script>
<script language="javascript" type="text/javascript" src="http://nb.mit.edu/embed_NB.js"></script>
</body>
</html>
