<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <!--[if lt IE 9]>
    <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
  <![endif]-->
  <link rel="stylesheet" href="./stylesheets/Mystyle.css">
  <script language="javascript" type="text/javascript" src="http://nb.mit.edu/embed_NB.js"></script>
</head>
<body>
<h3 id="dna-replication">DNA Replication</h3>
<hr />
<p>The double-helical structure of DNA immediately suggested to Watson and Crick a simple mechanism for the accurate duplication of the genetic information stored in DNA.</p>
<p>Each strand contains all of the information necessary to specify the sequence of its complementary strand.</p>
<p>Polymerization (always) involves the reaction of an incoming deoxynucleotide triphosphate (dNTP) with the 3’ hydroxy group of the pre-existing polynucleotide chain [<a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0018881">link</a>].</p>
<p>The reaction releases pyrophosphate (P-P) and forms a phosphodiester bond. The 3’ OH of the added dNTP becomes the new 3’ OH of the growing polynucleotide.</p>
<figure>
<img src="./img/replication.jpg" alt="Figure: DNA replication" /><figcaption><em>Figure: DNA replication</em></figcaption>
</figure>
<p>The sequence of a polynucleotide is always <strong>written in the 5’ to 3’ direction</strong>, just as the sequence of a polypeptide is always written from its N-terminus to its C-terminus (as we will see).</p>
<p>Both DNA and RNA polymerases synthesize nucleic acids by adding nucleotides to the 3’-OH end of a molecule.</p>
<embed width="420" height="345" src="http://www.youtube.com/v/WxrxPrTN9Gk" type="application/x-shockwave-flash"> </embed>

<p><strong>DNA replication</strong> is a complex process - the steps are mediated by various proteins. In the bacterium <a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1000344"><em>Escherichia coli</em></a> over 100 genes are involved in DNA replication and repair.</p>
<p>Replication begins at specific sequences along the DNA strand, known as origins of replication or origins for short.</p>
<p>Origin sequences are recognized by specific proteins. The binding of these proteins initiates the assembly of an origin recognition complex, an ORC.</p>
<p>In the laboratory, DNA is generally heated to 94-100°C in order to insure complete denaturation (strand separation). In the cell, various proteins act on the DNA to locally denature (unwind) the DNA to form a replication bubble.</p>
<p>A multiprotein complex will assemble at each end of the replication bubble, these structures are known as replication forks.</p>
<figure>
<img src="./img/realfork.jpg" alt="Figure: Replication forks" /><figcaption><em>Figure: Replication forks</em></figcaption>
</figure>
<p>Using a single replication origin and two replication forks moving in opposite directions, a growing <em>E. coli</em> can replicate its ~4,700,000 base pairs of DNA in ~40 minutes.</p>
<p>Each replication fork moves along the DNA synthesizing ~1000 base pairs of DNA per second</p>
<p>Synthesis is an accurate process; the polymerase makes about one error for every 10,000 base pairs it synthesizes. Most of these errors are quickly recognized, since they lead to the formation of a base mismatch.</p>
<p>When a mismatched base pair is formed, the DNA polymerase recognizes its presence, reverses and removes it (using an exonuclease activity), and than resynthesizes it, correctly.</p>
<p>This process is known as proof-reading; the proof-reading activity of the DNA polymerase complex reduces the total DNA synthesis error rate to ~1 error per 1,000,000,000 (10<sup>9</sup>) base pairs synthesized.</p>
<hr />
<p><strong>DNA replication and molecular machines:</strong></p>
<p>The process of nucleic acid replication involves a number of molecular machines, we will consider only one as an example - the clamp loader.</p>
<p>Once DNA replication begins it is important that the polymerase complex remains attached to the DNA.</p>
<p>This is accomplished by the assembly of a doughnut shaped protein, known as a sliding clamp, around the DNA double helix.</p>
<embed width="420" height="345" src="http://www.youtube.com/v/QMhi9dxWaM8" type="application/x-shockwave-flash"> </embed>

<p>The clamp is added to DNA by a protein known as a clamp loader [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3331839/?tool=pubmed">link</a>]. Once closed around the DNA the clamp diffuses freely along the length of the DNA molecule, but it cannot leave the DNA. The rest of the polymerase complex is attached to the clamp.</p>
<p><img src="./img/clampSide.gif" alt="clamp" /> <img src="./img/clampTOP.gif" alt="clamp" /></p>
<p><em>Figure: Molecular structure of the sliding clamp</em></p>
<p>This keeps the polymerase complex from drifting away from the DNA. The attached clamp-polymerase complex now assembles many nucleotides into a new DNA strand before it falls off.</p>
<p>DNA-dependent, DNA polymerases cannot start a new DNA chain de novo or from nothing. They can only add nucleotides to a pre-existing DNA (or RNA) strand. Why this is, evolutionarily, is not known, but it is a conserved (homologous) feature of the DNA polymerases involved in DNA replication.</p>
<p>When the DNA helix opens there are no pre-existing DNA strands for the DNA polymerase to extend.</p>
<p>So, how does synthesis start?</p>
<p>The answer is that a short RNA molecules, known as a primers, complementary to the two strands of the open DNA, are synthesized by a DNA-dependent, RNA polymerase - this enzyme is known as <a href="http://chem-mgriep2.unl.edu/replic/Primase.html">primase</a>.</p>
<p><img src="./img/primase1.jpg" alt="primase 1" /><br /> <img src="./img/primase2.jpg" alt="primase 2" /></p>
<p><em>Figure: DNA before primers are added via primase (top) and with primers (bottom)</em></p>
<p>DNA polymerase synthesizes DNA onto the ends of these primers in the standard 5’ to 3’ manner. These first primers define the leading strands of DNA synthesis.</p>
<p>As the DNA forks move, more DNA is unwound, and new primers have to be synthesized to insure replication of the “lagging strands”(<a href="http://www.biostudio.com/d_%20DNA%20Replication%20Coordination%20Leading%20Lagging%20Strand%20Synthesis.htm">watch this animation</a>: quite nice, although very “flat”). No new primers have to be synthesized on the leading strand.</p>
<p>Later in the DNA replication process, the RNA primers are digested away, replaced with DNA, and joined together by the enzyme DNA ligase to produce a single, uninterrupted replicated strand.</p>
<p><img src="./img/ReplicationFork1.jpg" alt="replication fork 1" /><br /> <img src="./img/ReplicationFork2.jpg" alt="replication fork 2" /></p>
<p>Eukaryotic cells can contain more than 1000 times the DNA found in a typical bacterial cell. Moreover, the eukaryotic DNA replication enzyme complex is much slower (about 1/20th as fast) than the prokaryotic system.</p>
<p>While a bacterial cell can replication is circular, ~3 x 10<sup>6</sup> base pair chromosome in about 1500 seconds using a single origin of replication. To replicate their DNA eukaryotic cells use multiple origins of replication, scattered along the length of each chromosome.</p>
<p>Another topological complexity is associated with the replication (and segregation) of closed circular DNA molecules, as well as extremely long linear DNA molecules.</p>
<embed width="420" height="345" src="http://www.youtube.com/v/EYGrElVyHnU" type="application/x-shockwave-flash"> </embed>

<p>http://www.youtube.com/watch?v=EYGrElVyHnU&amp;feature=player_embedded</p>
<p>The replication of DNA unwinds the DNA, and this unwinding requires topoisomerases to allow replication to continue. </p>
<p>A specific molecular machine acts when replication forks “crash” into one another.</p>
<p>It resolves the strands and links them, leading to a single double stranded DNA molecule per chromosome.</p>
<p>So even though they may contain 1000 times more DNA than <em>E. coli</em>, DNA replication can still be completed in as little as 10 minutes.</p>
<p>Below is a video of the complex process of DNA replication (something to look forward to if anyone goes on to take Molecular Biology!)</p>
<embed width="420" height="345" src="http://www.youtube.com/v/4jtmOZaIvS0" type="application/x-shockwave-flash"> </embed>

<hr />
<p><strong>Questions to answer</strong></p>
<ol type="1">
<li>What defines the directionality (polarity) of a polynucleotide chain?</li>
<li>How does the DNA polymerase recognize a mismatched base pair?</li>
<li>Why do you need to denature (melt) the DNA double-helix to copy it?</li>
<li>How does the DNA polymerase complex know where to start replicating DNA?</li>
<li>Make a cartoon of a prokaryotic chromosome, indicate where replication starts and stops. Now make a cartoon of a eukaryotic chromosomes. What specific issues are associated with the termination of DNA in each case?</li>
<li>List all of the unrealistic components in the replication video?</li>
<li>Is an RNA primer needed to make an mRNA, do you think?</li>
<li>Why is only a single RNA primer needed to synthesize the leading strands, but multiple primers are needed to synthesize the lagging strands?</li>
<li>During the replication of a single DNA molecule, how many leading and lagging strands are there?</li>
</ol>
<p><strong>Questions to ponder</strong></p>
<ul>
<li>Assume that there is a mutation that alters the proof-reading function of the DNA polymerase complex - what will happen to the cell?</li>
<li>Explain why, in the absence of the clamp, the polymerase could diffuse away from the DNA? Why might this be “bad”?</li>
<li>How do you think the clamp is removed?</li>
</ul>
<hr />
</body>
</html>
