<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <!--[if lt IE 9]>
    <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
  <![endif]-->
  <link rel="stylesheet" href="./stylesheets/Mystyle.css">
  <script language="javascript" type="text/javascript" src="http://nb.mit.edu/embed_NB.js"></script>
</head>
<body>
<h3 id="making-polypeptides-and-proteins">Making polypeptides (and proteins)</h3>
<hr />
<p>DNA is more or less inert, it acts simply to store information. For that information to be used and useful, it must be transformed first into <a href="http://en.wikipedia.org/wiki/RNA">RNA</a>; this process is known as <a href="http://en.wikipedia.org/wiki/Transcription_%28genetics%29">transcription</a>.</p>
<p>It is called transcription because there is no “change in language”; information is encoded in the same way, through the sequence of bases in both DNA and RNA.</p>
<p>Transcription is mediated by DNA-dependent, RNA polymerase, an enzyme that builds an RNA molecule using a DNA molecule. </p>
<p>The larger the <a href="http://en.wikipedia.org/wiki/Genome">genome</a>, the more information it can, at least in theory, contain. </p>
<p>The organism with the largest known genome is the plant, <em>Paris japonica</em> (<a href="http://www.sciencedaily.com/releases/2010/10/101007120641.htm">Pellicer et al, 2010</a>), with an estimated size of ~150,000 x 10<sup>6</sup> (millions of) base pairs, which compares to the ~3,200 x 10<sup>6</sup> base pairs of humans (which encodes approximately 25,000 genes). </p>
<p>Smaller genomes are found in bacteria and archaea, but the smallest genomes occur in obligate parasites and endosymbionts (can you guess why?). </p>
<p>For example, the bacterium <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=7569993&amp;dopt=Abstract"><em>Mycoplasma genitalium</em></a>, the cause of non-gonococcal urethritis, contains ~0.580 x 10<sup>6</sup> base pairs of DNA, which encodes ~500 distinct genes. It has the smallest genome found for an organism that can be grown independently.  Using mutagenesis studies, Lewis et al (<a href="http://www.pnas.org/content/103/2/425.full">2006</a>) found that 382 of its genes are essential, of which ~28% had unknown functions. </p>
<p>The obligate endosymbiont, <em>Carsonella ruddii</em> has 159,662 (~0.16 x 10<sup>6</sup>) base pairs of DNA encoding &quot;182 ORFs (<em>open reading frames = genes</em>), 164 (90%) overlap with at least one of the two adjacent ORFs&quot; (Nakabachi et al, 2006).</p>
<p>  <img src="./img/Carsonella.jpg" alt="Relationship between genome sizes and GC content of 358 complete genomes from Bacteria and Archaea. In the insert, the tubular structures are Carsonella living inside a cell of a phloem sap-feeding insect Pachypsylla venusta - the host cell&#39;s nucleus is the larger blue oval" /></p>
<p>Figure:Relationship between genome sizes and GC content of 358 complete genomes from Bacteria and Archaea. In the insert, the tubular structures are <em>Carsonella</em> living inside a cell of a phloem sap-feeding insect <em>Pachypsylla venusta</em> - the host cell's nucleus is the larger blue oval</p>
<hr />
<p><strong>Locating information within DNA:</strong></p>
<p>The DNA molecules that contain genes can be thousands to millions of base pairs in length. More to the point, a single DNA molecule can contain hundreds to thousands of genes. </p>
<figure>
<img src="./img/ecoli-dna.jpg" alt="E.coli DNA released from the cell" /><figcaption><em>E.coli</em> DNA released from the cell</figcaption>
</figure>
<p>To access the information stored in a gene, it is necessary to find the sequence of the gene so that it can be transcribed (copied) into RNA. This sequence has to be recognized and distinguished from all of the other DNA sequences present within the cell. </p>
<p>This is accomplished by using a two-component system.  The first part of this system are specific nucleotide sequences within the DNA (very much like the origin of replication sequences we discussed before.)</p>
<p>These sequences provide a molecular address that can be used to identify the specific region of a specific strand of the DNA to be transcribed.  The set of sequences involved in identifying the region of a gene to be transcribed is known as the gene's regulatory region. </p>
<p>The regulatory region of a gene can be simple and relatively short or complex and extremely long. In some human genes, the gene's regulatory region is spread over thousands of base-pairs of DNA, located both &quot;up-stream&quot; and &quot;down-stream&quot; of the coding region.</p>
<p>You can carry out a simple mathematical exercise, how long would a nucleotide sequence have to be, to be unique within 3,200,000,000 base pairs? </p>
<figure>
<img src="./img/DNA_animation_white.gif" alt="Transcription factors (purple and blue proteins) bound to DNA" /><figcaption>Transcription factors (purple and blue proteins) bound to DNA</figcaption>
</figure>
<p>Regulatory DNA sequences are recognized by proteins that bind to DNA in a sequence specific manner; these proteins are known as transcription factors because they regulate transcription.  </p>
<p>Specific regulatory sequences are bound by specific transcription factors, and it is common that multiple transcription factors are used together (combinatorially) to recognize and regulate the expression of specific genes.</p>
<p>A specific transcription factor can act either positively or negatively.  </p>
<p>Positively-acting transcription factors, known as activators, stimulate transcription. </p>
<p>Negatively-acting transcription factors, known as repressors, suppress transcription.</p>
<p>Sometimes a transcription factor that acts positively at one DNA regulatory site will act negatively at another, depending upon its interactions with other proteins that bind near by, or that associate with the transcription factor. Transcription factors are &quot;context dependent&quot;.  </p>
<figure>
<img src="./img/transcription_factors.gif" alt="Transcription factors (purple proteins) bound to DNA" /><figcaption>Transcription factors (purple proteins) bound to DNA</figcaption>
</figure>
<p>DNA-binding transcription factors recognize specific DNA sequences by interacting with the surfaces of the base pairs visible in the major and minor grooves of the DNA helix.</p>
<figure>
<img src="./img/MajorMinorGrooves.jpg" alt="The major and minor grooves of DNA" /><figcaption>The major and minor grooves of DNA</figcaption>
</figure>
<p>A single base pair change in a gene's regulatory region can profoundly alter the efficiency or timing of transcription.</p>
<hr />
<p>The binding of transcription factors recruits or inhibits the recruitment of an enzyme complex, the DNA-dependent, RNA polymerase, to the DNA.</p>
<p><img src="./img/polymeraseLoading1.jpg" />]_<img src="./img/polymeraseLoading2.jpg" /></p>
<p>This RNA polymerase synthesizes RNAs.</p>
<p>Where RNA polymerase starts transcribing RNA is known as the transcription start site. Where it falls off the DNA, and so stops transcribing RNA, is known as the transcription termination site.</p>
<p>As transcription initiates, the RNA polymerase moves away from the transcription start site. </p>
<p>Once the RNA polymerase complex moves far enough away (clears the start site), there is room for a new polymerase complex to associate with the DNA, through interactions with transcription factors. </p>
<p>The efficiency of RNA polymerase binding and activation determines the rate at which RNA molecules are produced. </p>
<p>  <a href="http://www.rcsb.org/pdb/molecules/pdb40_1.html"><img src="./img/RNA_polymerase_and_DNA.jpg" /></a></p>
<p><em>RNA polymerase bound to DNA, click the image to learn more about the process of transcription</em></p>
<hr />
<p><strong>Questions to answer</strong></p>
<ol type="1">
<li>Make a diagram that illustrates how genes could &quot;overlap&quot;. Explain what your diagram looks like.</li>
<li>Present a model for how the binding of a transcription determines which DNA strand will be transcribed. </li>
<li>Why does binding of combinations of transcription factors increase the selectivity of the regulatory system.</li>
<li>Describe the possible effects of a mutation that alters the DNA-binding specificity of a particular transcription factor.</li>
<li>Describe the possible effects of a mutation that alters the sequence normally recognized by a specific transcription factor.</li>
<li>Describe the possible effects of a mutation that reduces the activity of DNA-dependent, RNA polymerase.</li>
<li>Consider a particular gene, what factors are likely to influence the length of its regulatory region?</li>
</ol>
<p><strong>Questions to ponder</strong></p>
<ul>
<li>What factors might drive he evolution of overlapping genes?</li>
<li>Explain why parasites and endosymbionts can survive with so few genes.</li>
</ul>
<hr />
<p>When the information is to be used to produce a polypeptide, the RNA is known as a messenger RNA or <a href="http://en.wikipedia.org/wiki/Messenger_RNA">mRNA</a>. The information in the mRNA will then be translated from a nucleotide into a polypeptide sequence. This process involves a large macromolecular complex, the ribosome, and various accessory factors.</p>
<p>The sequence of nucleotides in DNA is related to the final sequence of the polypeptide by the <a href="http://en.wikipedia.org/wiki/Genetic_code">genetic code</a>.</p>
<p>The genetic code is not the information itself, but the <a href="http://en.wikipedia.org/wiki/Algorithm">algorithm</a> by which nucleotide sequences encode polypeptide sequences. The code consist of codons, which are nucleotides, read three at a time without spaces.</p>
<p>Since there are 4 nucleotides, there are 4<sup>3</sup> (or 64) possible codons. Since all organisms use the same 20 amino acids, the code is redundant - certain amino acids are encoded for by more than one codon.</p>
<p>There are also three codons, UAA, UAG and UGA that encode “stops” or periods.</p>
<p><img src="./img/Figure_09_04_02.jpg" alt="Genetic code algorithm" width="400" height="350"></p>
<p><a href="http://en.wikipedia.org/wiki/Genetic_code"><em>Figure: The genetic code algorithm</em></a></p>
<p>In addition to the common 20 amino/imino acids, there are times when other amino acids are used, in particular selenocysteine and pyrrolysine.</p>
<p>In the case of selenocysteine, the amino acid is encoded by a stop codon, UGA, that is in a particular context within the RNA [see <a href="http://www.ncbi.nlm.nih.gov/pubmed/8811175">here</a> for more details - totally optional].</p>
<p><a href="http://en.wikipedia.org/wiki/Selenocysteine"><img src="./img/selenocysteine-2D-skeletal.png" alt="selenocysteine" /></a></p>
<p><a href="http://en.wikipedia.org/wiki/Selenocysteine">Selenocysteine</a></p>
<p>In the case of pyrrolysine, coding also involves a stop codon. In this case, a gene encodes a special tRNA that recognizes the normal stop codon UAG (and is charged with pyrrolysine - see below).</p>
<p><a href="http://en.wikipedia.org/wiki/Pyrrolysine"><img src="./img/Pyrrolysine.png" alt="pyrrolysine" />pyrrolysine</a></p>
<p>There are a number of hypotheses on the origin of the genetic code, from the frozen accident model, to one in which specific interactions between RNAs and amino acids play an important role.</p>
<p>What these unusual amino acids, and other variations in the genetic code (<a href="http://www.nature.com/nrg/journal/v2/n1/abs/nrg0101_049a_fs.html;jsessionid=AD81AAA41AE9E5E43F0D6F3EE71A6CE3">see here</a>), illustrate is that evolutionary mechanisms can change the genetic code. Since it does not appear to be “predetermined”, the general conservation of the genetic code among organisms is seen as strong evidence that all organisms (even the ones with minor variations in the genetic code) are derived from a single common ancestor.</p>
<p>It appears that the genetic code is a homologous trait between all organisms.</p>
<hr />
<p>Translation involves a complex cellular organelle, the <a href="http://en.wikipedia.org/wiki/Ribosome">ribosome</a>, which together with a number of accessory factors, including transfer or tRNAs, reads the code in an mRNA and produces the appropriate polypeptide.</p>
<figure>
<img src="./img/Information.jpg" alt="dna" /><figcaption>dna</figcaption>
</figure>
<p>[can’t stop yourself? <a href="translation-details.htm">click here</a> for a more detailed description of translation.]</p>
<p>In general (but not always), the first 5’ AUG in the mRNA, which encodes the amino acid methionine, serves to mark the start of translation.</p>
<figure>
<img src="./img/translation_1.gif" alt="Translation" /><figcaption>Translation</figcaption>
</figure>
<p>Ribosomes are composed of roughly equal amounts of ribosomal (rRNAs) and ribosomal polypeptides. An active ribosome is composed of a small and a large ribosomal subunit.</p>
<p>The complete ribosome has a molecular weight of ~3 x 10<sup>6</sup> daltons. A evolutionarily conserved catalytic rRNA, a ribozyme, lies at the heart of the ribosome - it catalyzes the addition of amino acids to the growing polypeptide chain.</p>
<p>The cytoplasm of cells is packed with ribosomes. In a rapidly growing bacterial cell, approximately 25% of the total cell mass is ribosomes.</p>
<p><a href="http://www.scripps.edu/pub/goodsell/gallery/patterson.html"><img src="./img/ribosome%20in%20situ.jpg" /></a></p>
<p><em>Figure: A David S. Goodsell image.</em></p>
<p>Although structurally similar, there are characteristic differences between the ribosomes of bacteria and eukaryotes.</p>
<p>This is important from a practical perspective. For example, a number of antibiotics selectively <a href="http://www.bmb.leeds.ac.uk/mbiology/ug/ugteach/icu8/antibiotics/protein.html">inhibit translation by bacterial</a>, but not eukaryotic ribosomes.</p>
<p>Both chloroplasts and mitochondria have ribosomes of the bacterial type.</p>
<p>This is yet another piece of evidence that chloroplasts and mitochondria are descended from bacterial endosymbionts.</p>
<p>The small and large subunits of the ribosome remain separate until they find an mRNA.</p>
<figure>
<img src="./img/ribosomeAssembly.jpg" alt="Ribosome" /><figcaption>Ribosome</figcaption>
</figure>
<p>Together with accessory factors, they associate with the mRNA and assemble into a functional ribosome, which then translates the mRNA.</p>
<p>When the ribosome reaches the end of the region of the RNA that encodes the polypeptide (defined by a stop codon), it is released, disassembles and is ready to <a href="translation-details.htm">start another cycle</a>.</p>
<p>Key translation accessory factors are the transfer RNAs or tRNA.</p>
<p>These are small, L-shaped RNAs. There are specific tRNAs for each amino acid. For example, a tRNA specific for phenylalanine would be written tRNA<sup>Phe</sup>.</p>
<figure>
<img src="./img/tRNA.jpg" alt="tRNA" /><figcaption>tRNA</figcaption>
</figure>
<p>Enzymes, amino acyl tRNA synthetases, recognize specific tRNAs and catalyze the attachment of the appropriate amino acid to the tRNA’s acceptor stem.</p>
<p><a href="http://www.rcsb.org/pdb/static.do?p=education_discussion/molecule_of_the_month/pdb16_1.html"><img src="./img/aminoAcylSynthetase.jpg" /></a></p>
<p>A codon in an mRNA is recognized by the anticodon in the tRNA molecule.</p>
<p><strong>Initiating translation:</strong> To make the correct polypeptide, the ribosome must start translating an mRNA at a specific point, the start codon.</p>
<p>In eukaryotes, the initial amino acid of the polypeptide is almost always encoded by the first 5’ AUG in the mRNA, its position determines the “reading frame” of the mRNA (with each subsequent triplet being recognized by the appropriate amino acid charged tRNA. Similarly, the end of the polypeptide is marked by a stop codon(UGA, UAA or UAG).</p>
<p><a href="http://www.ncbi.nlm.nih.gov/Structure/mmdb/mmdbsrv.cgi?form=6&amp;db=t&amp;Dopt=s&amp;uid=12130"><img src="./img/EuTF_tRNA%20Complex.jpg" /></a>A</p>
<p><em>Figure: tRNA associated with the elongation factor TU (EF-TU)</em></p>
<p>Accessory factors are associated with translation initiation, elongation and termination.</p>
<p>The mRNA moves through the ribosome, bringing one codon after another into place.</p>
<p>Each codon is recognized by an amino acid-charged tRNA.</p>
<p><img src="./img/Figure_09_04_03.jpg" alt="Translation Steps" width="250" height="420"></p>
<p>This is a stochastic process , on average many amino-acid charged tRNAs will collide with the ribosome before the “right” one (determined by the codon-anti-codon interaction), hits and binds productively.</p>
<p>Once in place, the ribosome catalyzes the formation of a peptide bond and the transfer of the growing or <strong>nascent polypeptide</strong> to the newly arrived amino acid-charged tRNA and the release of the now uncharged tRNA.</p>
<p><a href="http://www.johnkyrk.com/DNAtranslation.html">Here is a nice translation tutorial</a></p>
<p>(What is (seriously) unrealistic about this tutorial?)</p>
<p>The process of translation uses energy (in the form of coupled reactions) both to move the mRNA through the ribosome and to form the peptide bonds.</p>
<p>Normally, there are no tRNAs that recognize stop codons, so when the ribosome reaches a stop codon, it stalls as it waits for a charged tRNA which will never arrive.</p>
<p>Instead, a polypeptide known as a release factor, which looks very much like a tRNA, can bind instead. This leads to the release of the polypeptide from the ribosome, and the disassembly of the translation complex.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/Structure/mmdb/mmdbsrv.cgi?form=6&amp;db=t&amp;Dopt=s&amp;uid=11752"><img src="./img/releaseFactor.jpg" /></a></p>
<hr />
<embed width="420" height="345" src="http://www.youtube.com/v/TfYf_rPWUdY" type="application/x-shockwave-flash"> </embed>

<hr />
<p><strong>Questions to answer</strong></p>
<ol type="1">
<li>What types of molecules can DNA encode directly? How about indirectly?</li>
<li>Aside from mRNA, which RNAs are involved in protein synthesis?</li>
<li>How, in the most basic terms, do different tRNAs differ from one another?</li>
<li>How many different tRNA-amino acid synthetases do you think there must be in a cell?</li>
<li>What could happen if a ribosome started translating an mRNA at the “wrong” place?</li>
<li>Why doesn’t the presence of release factor in the cell cause the premature termination of translation at non-stop codons?</li>
<li>Which molecule(s) actually do the translating?</li>
</ol>
<p><strong>Questions to ponder</strong></p>
<ul>
<li>What does it mean to say the genetic code is an algorithm?</li>
<li>Do variations in the genetic code argue for against a common ancestor for life on earth?</li>
<li>What is meant when people call the genetic code a “frozen accident”?</li>
</ul>
<hr />
</body>
</html>
