<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <!--[if lt IE 9]>
    <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
  <![endif]-->
  <link rel="stylesheet" href="./stylesheets/Mystyle.css">
  <script language="javascript" type="text/javascript" src="http://nb.mit.edu/embed_NB.js"></script>
</head>
<body>
<p>###. Nucleic Acids &amp; Heredity</p>
<hr />
<p>One of the amazing facts associated with Darwin and Wallace’s original evolutionary hypothesis was their complete lack of a coherent understanding of genetic mechanism.</p>
<p>While it was clear that organisms varied, and that part of that variability was inherited, the mechanism by which genetic information was stored and transmitted was completely unclear.</p>
<p>Beginning with the work of Gregor Mendel in the 19th century, which was rediscovered and built upon beginning in the early part of the 20th century, it became obvious that hereditary “factors” - which became known as <strong>genes</strong>, were present as discrete objects that could be passed (generally) unchanged from generation to generation.</p>
<p>Specific genes could exist in different forms, known as alleles. In higher organisms, two copies of each gene were present, one copy inherited from the maternal parent, the other from the paternal parent.</p>
<p>It was, however, unclear what genes were made of, and how they stored information.</p>
<hr />
<p><strong>Discovering how nucleic acids store genetic information</strong></p>
<p>Nucleic acids were first isolated as part of the phosphorus-rich compound nuclein from pus and sperm by Friedrich Miescher.</p>
<p><a href="http://www.dnaftb.org/15/bio.html"><img src="./img/Miescher.jpg" alt="Figure&quot;: Mieschner and his family" /></a><br /> <em>Figure: Miescher and family. Click the image to learn more</em></p>
<p>Later nuclein was resolved into an acidic component, nucleic acid,and a basic component, primarily proteins known as histones.</p>
<p>Eukaryotic cells contain a distinct structure, the nucleus, where nuclein was localized.</p>
<p>During cell division, the nucleus appeared to be replaced by densely staining bodies, known as chromosomes.</p>
<embed width="420" height="345" src="http://www.youtube.com/v/c6TlHINXSj8" type="application/x-shockwave-flash"> </embed>
<p><br /> In 1887 <a href="http://www.ijdb.ehu.es/web/paper.php?doi=1627480">Edouard van Beneden</a> (below) reported that the number of chromosomes was a constant for each species.</p>
<figure>
<img src="./img/vanbeneden.jpg" alt="Edward van Bendeen" /><figcaption>Edward van Bendeen</figcaption>
</figure>
<p>Different species had different numbers of chromosomes.</p>
<figure>
<img src="./img/Ch1Ct1.gif" alt="Figure:Chromosome number variation across species" /><figcaption><em>Figure:Chromosome number variation across species</em></figcaption>
</figure>
<p>In 1902, Walter Sutton published his observation that chromosomes obey Mendel’s rules of inheritance.</p>
<p>This strongly suggested that Mendel’s genetic factors were associated with the chromosomes.</p>
<p>That hereditary information was localized to nuclei was further suggested by a number of observations, particularly the transplantation experiments carried out by Joachim Hammerling in the 1930’s.</p>
<p><a href="http://arjournals.annualreviews.org/doi/full/10.1146/annurev.arplant.49.1.173?maxtoshow=&amp;issue=1&amp;RESULTFORMAT=&amp;volume=49&amp;HITS=&amp;FIRSTINDEX=&amp;searchid=QID_NOT_SET&amp;fulltext=acetabularia&amp;hits=&amp;journalcode=arplant&amp;cookieSet=1"><img src="./img/acetabularia2.jpg" alt="Acetabularia" /></a></p>
<p>He showed that the information needed to control the morphology of the giant unicellular green alga <em>Acetabularia</em>, the mermaid’s wineglass (above), was located within the region of the cell that contained the nucleus.</p>
<embed width="420" height="345" src="http://www.youtube.com/v/tl5KkUnH6y0" type="application/x-shockwave-flash"> </embed>

<hr />
<p>The proof that DNA was the genetic material came from a series of experiments initiated in the 1920s by Fred Griffith.</p>
<p>He was studying two strains of the bacterium <em><a href="http://www.cdc.gov/ncidod/dbmd/diseaseinfo/streppneum_t.htm">Streptococcus pneumoniae</a>.</em></p>
<p>The two strains differed in the nature of the colonies they formed; in one case the colonies were smooth, in the other case, the colonies were rough.</p>
<p><img src="./img/ColoniesSmooth.jpg" alt="Smooth colonies" /> <img src="./img/ColoniesRough.jpg" alt="rough colonies" /></p>
<p><em>Figure: Smooth (left) and rough (right) Streptococcus pneumoniae</em></p>
<p>He found that when cells of the smooth colony <strong>S</strong> strain were injected into mice the mice quickly sickened and died.</p>
<p>However, during extended cultivation in vitro, the <strong>S</strong> strain sometimes gave rise to rough <strong>(R)</strong> colonies.</p>
<p>Mice injected with <strong>R</strong> strain bacteria did not get sick.</p>
<p>Likewise, mice injected with heat killed S strain bacteria also did not get sick. BUT, weirdly enough, mice co-injected with the living <strong>R</strong> and dead <strong>S</strong> bacteria got sick and died! From these dying mice, he isolated a new pathogenic smooth strain, which he termed <strong>S-II</strong>.</p>
<figure>
<img src="./img/Griffith.jpg" alt="Figure: Griffith&#39;s Experiments" /><figcaption><em>Figure: Griffith's Experiments</em></figcaption>
</figure>
<p>His hypothesis was that a non-living component derived from the dead <strong>S</strong> bacteria had “transformed” the avirulent (benign) <strong>R</strong> strain into the virulent <strong>S-II</strong> strain.</p>
<hr />
<p><strong>The molecular basis of transformation:</strong></p>
<p>In 1944, these studies were followed up by <a href="http://profiles.nlm.nih.gov/CC/A/A/N/V/">Oswald Avery, Colin McLeod and Maclyn McCarty</a>. They set out to use Griffith’s assay to isolate the transforming principle responsible for turning R into S strains.</p>
<p>Their approach was to make cell extracts; they ground up cells and isolated various components (such as proteins, nucleic acids, carbohydrates, lipids).</p>
<p>They then digested (i.e., enzymatically destroyed) these extracts with various enzymes and asked whether the transforming principle was still intact.</p>
<p>Treating cellular extracts with proteases, which degrade proteins lipases, which degrade lipids, or RNAases, which degrade RNAs had no effect on transformation.</p>
<p>In contrast, treatment of the extracts with DNAase, which degrades DNA, destroyed the activity.</p>
<p>Furthermore, purification of the transforming substance suggested that it had the properties of DNA and subsequent studies confirmed this conclusion.</p>
<p>DNA isolated from R strain bacteria did not produce S-strain bacteria, whereas DNA from S strain bacteria could transform R into S.</p>
<p>They concluded that DNA derived from S cells contains the information required for the conversion – it is a gene or part of one.</p>
<figure>
<img src="./img/nucleusDNA.jpg" alt="The nucleus and DNA" /><figcaption>The nucleus and DNA</figcaption>
</figure>
<hr />
<p><strong>Horizontal gene transfer:</strong></p>
<p>The process of transformation, the picking up and integrating of DNA from the environment (as occurred during Griffith’s experiments) is thought to have played a key role in early evolution and in the rapid evolution of bacteria today.</p>
<figure>
<img src="./img/insertion.gif" alt="Insertion of DNA" /><figcaption>Insertion of DNA</figcaption>
</figure>
<p><em>Figure: Bacteria can integrate foreign DNA into their genome.</em></p>
<p>In the modern world, the rapid spread of antibiotic resistance is due to horizontal gene transfer. Genes encoding factors involved in antibiotic resistance are released from dying bacteria, and can be acquired by their neighbors.</p>
<p>When we analyze the total DNA of an organism, its genome, we find evidence for horizontal gene transfer, particularly within the bacteria and archaea.</p>
<hr />
<p><strong>Questions to answer</strong></p>
<ol type="1">
<li>Based on the data presented, is there a correlation between the number of chromosomes and the complexity of an organism?</li>
<li>Would you expect that closely related species would have the same number of chromosomes?</li>
<li>What caused the change from S to R strains in culture?</li>
<li>In Griffith’s study, he found that dead smooth <em>S. pneumoniae</em> could transform living rough strains of <em>S. pneumoniae</em> when co-injected into a mouse (<em>in vivo</em>). Would another species of dead bacteria give the same result?</li>
<li>How would Hammerling’s observations have been different if the hereditary information was localized in the cytoplasm?</li>
</ol>
<p><strong>Questions to ponder</strong></p>
<ul>
<li>How might horizontal gene transfer confuse molecular phylogenies?</li>
<li>Where did the original genes come from?</li>
</ul>
<hr />
</body>
</html>
